features/time.yml.dist

```diff
@@ -14,7 +14,6 @@ status:
     safari: "10"
     safari_ios: "10"
 compat_features:
-  # ⬇️ Same status as overall feature ⬇️
```

Comment:
These changes are because of https://github.com/mdn/browser-compat-data/pull/23638.